tests/settings.py

```diff
@@ -20,3 +20,5 @@
 ELASTIC_SEARCH.update({
     "index_prefix": "test",
 })
+
+AUTOCLASSIFY_JOBS = True
```

Comment:
I think we should make the default in config/settings.py be True instead perhaps?